examples/light-switch-app/nrfconnect/CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.20.0)
+
+get_filename_component(CHIP_ROOT ${CMAKE_CURRENT_SOURCE_DIR}/third_party/connectedhomeip REALPATH)
+get_filename_component(NLIO_ROOT ${CHIP_ROOT}/third_party/nlio/repo/include REALPATH)
```

Comment:
I think you can remove this line, as I moved NLIO_ROOT to generic nrfconnect cmake file in the last week, so all examples should be able to access it without direct include.